live-examples/js-examples/arraybuffer-slice.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js" class="language-js">// create an ArrayBuffer with a size in bytes
+var buffer = new ArrayBuffer(16);
+
+var int32View = new Int32Array(buffer); // create the view
+int32View[2] = 42;
+var lastHalfView = new Int32Array(buffer.slice(8,16));
```

Comment:
Because the 2nd parameter is optional, this would also work if you did just slice(8), no? This could therefore be a little misleading. I'd include two examples, one with and one without the second parameter, to make it clear that it is optional, and show what it does.

Reply:
Fair point. OK to remove the second parameter in this case I think. Again we are working with the restriction of having a single console.log and one interactive example per page, so not sure we can include two examples.

Reply:
I think having an example including the second parameter that actually needs it would be better. Maybe do a slice to return the middle 4 bytes?

Reply:
Yes - makes sense.

Reply:
BTW The limitation I'm coming up against is that it's difficult to console.log the whole array as that involves either looping (multiple console.logs) or some extra code to somehow 'flatten' first.